src/main/java/org/openrewrite/staticanalysis/CatchClauseOnlyRethrows.java

```diff
@@ -88,6 +87,14 @@ public J.Try visitTry(J.Try tryable, ExecutionContext ctx) {
                 }));
             }
 
+            private boolean hasWiderExceptionType(J.Try.Catch aCatch, J.Try.Catch next) {
+                if (next.getParameter().getType() instanceof JavaType.MultiCatch) {
+                    JavaType.MultiCatch multiCatch = (JavaType.MultiCatch) next.getParameter().getType();
+                    return multiCatch.getThrowableTypes().stream().anyMatch(alt -> TypeUtils.isAssignableTo(alt, aCatch.getParameter().getType()));
```

Comment:
We may want to use a foreach loop here as this recipe potentially gets ran a lot